.github/workflows/customer-service.yml

```diff
@@ -0,0 +1,27 @@
+name: customer-service-pipeline
+on:
+  push:
+    paths:
+      - 'customer-service/**'
+    branches: [ develop ]
+  pull_request:
+    paths:
+      - 'customer-service/**'
+    branches: [ develop ]
+
+jobs:
+  build-graalvm-ce:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: setup-graalvm-ce
+        uses: rinx/setup-graalvm-ce@v0.0.5
+        with:
+          graalvm-version: "20.2.0"
+          java-version: "java11"
+          native-image: "true"
+      - name: Build and test with Maven
+        run: |
+          cd customer-service
+          mvn -Dmaven.repo.local=.maven-m2 -Pnative clean verify
```

Comment:
Can you use the provided build script `build-native.sh` here? I think it would be easier to maintain only one build script in the whole process.

Reply:
yes good point, ill change it